public/index.html

```diff
@@ -7,6 +7,17 @@
     <meta name="theme-color" content="#000000" />
     <meta name="description" content="Web site created using create-react-app" />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <script src="https://apis.google.com/js/api.js"></script>
+    <script>
+      function start() {
+        gapi.client
+          .init({
+            apiKey: 'AIzaSyB8VGjAtysqiUWnHjaitKDMi4Kp9VHWKe0',
+          })
+          .then(() => {});
+      }
+      gapi.load('client', start);
```

Comment:
Be careful with your API key, please don't use your API keys directly on your code. Please take a look at [this article](https://medium.com/@soni.dumitru/keeping-your-api-keys-secret-with-dotenv-b66aa05fdf71).

```suggestion
      gapi.load('client');
```